src/bindings/bnd_extensions.cpp

```diff
@@ -669,6 +678,27 @@ bool BND_ONXModel::ReadTest(std::wstring path)
 }
 
 
+// --------------------- Iterator helpers ------- //
+#if defined(ON_PYTHON_COMPILE)
+namespace py = pybind11;
+
+template <typename IT, typename ET>
+struct PyBNDIterator {
+  PyBNDIterator(const IT table, py::object ref)
+    : seq(table), ref(ref) {}
+
+  ET next() {
+    if(index>=seq.Count()) throw py::stop_iteration();
+    return const_cast<IT>(seq).FindIndex(index++);
```

Comment:
This isn't going to work, I should find a better way to do this. At least for the object table I should wrap ModelObjectAt for BND_FileObject instead of BND_CommonObject